src/js/common/index.js

```diff
@@ -9,3 +9,13 @@ require('./utils/sticky-action-box.js');
 
 // New navigation menu
 require('../legacy/mega-menu.js');
+
+// Disable scrolling on input[type="number"] because it was changing the value unexpectedly
+// NOTE: This also just stops the user from scrolling when hovering over a number
+//  input that has focus. This isn't ideal, but until a solution is found to that,
+//  this is what we've got.
+document.addEventListener('mousewheel', (event) => {
```

Comment:
It looks like [`wheel` should be used instead of `mousewheel`](https://developer.mozilla.org/en-US/docs/Web/Events/mousewheel).